examples/postgres/test_all

```diff
@@ -1,7 +1,22 @@
 #!/bin/sh
 set -e
 
-export DATABASE_URL="postgres://localhost/diesel_examples"
+set -a
+. ../../.env || true
+DATABASE_URL=${DATABASE_URL-${PG_DATABASE_URL}}
+set +a
+
+trap finish EXIT
+
+function finish {
+  # Make sure when we're done, we leave behind an empty database. Otherwise,
+  # at the time of writing, running diesel_tests stumbles as it tries to run
+  # conflicting migrations on an overlapping schema.
+  #
+  # TODO: Find a better way to avoid mixing of resources across tests.
+  ../../bin/diesel database drop
+  ../../bin/diesel database touch
```

Comment:
Unfortunately, all of that juggling environment variables makes that not-so-easy. We could introduce yet another set of env vars (`{PG,MYSQL}_EXAMPLE_DATABASE_URL`?) and use that. Or try to change the database ad-hoc, but trying to do that in bash is going to be frail.
Or, not run those examples normally. However, once you did so manually, you'd run into the same problems again, so I'd rather avoid that option. Any preference on your end?

Also, any pointers on how I can investigate the result of `infer_schema!("dotenv:SQLITE_DATABASE_URL");` to track down the other spurious failure I mentioned?